osbrain/tests/test_proxy.py

```diff
@@ -57,7 +57,7 @@ def test_wrong_nameserver_address(timeout):
     t0 = time.time()
     with pytest.raises(TimeoutError):
         locate_ns('127.0.0.1:1234', timeout=timeout)
-    assert timeout <= time.time() - t0 <= timeout + 1.
+    assert timeout <= time.time() - t0 <= timeout + 2.
```

Comment:
Was this failing with the previous timeout?

Reply:
At least on my machine, yes. :smile:

Reply:
Turns out 1.2 is enough, but with just 1, the test will fail everytime, both on my machine and appveyor